calico/felix/dispatch.py

```diff
@@ -226,13 +230,35 @@ def _calculate_update(self, ifaces):
 
             if not use_root_chain:
                 # Add a default drop to the end of the leaf chain.
-                from_upds.append("--append %s --jump DROP" % disp_from_chain)
-                to_upds.append("--append %s --jump DROP" % disp_to_chain)
+                # Add a default drop to the end of the leaf chain.
+                from_upds.extend(
+                    self.config.plugins["iptables_generator"].drop_rules(
```

Comment:
Feels a bit long-winded/error-prone to have `self.config.plugins["iptables_generator"]` everywhere, can you either take a copy of it from the constructor, or introduce a property:
```
@property
def _ipt_generator(self):
    return self.config.plugins["iptables_generator"]
```